skrub/_dataframe/_common.py

```diff
@@ -781,7 +781,7 @@ def _to_datetime_polars(col, format, strict=True):
         # differently.
         format = format.replace(".%f", "%.f")
         return col.str.to_datetime(format=format, strict=strict)
-    except pl.ComputeError as e:
+    except (pl.exceptions.ComputeError, pl.exceptions.InvalidOperationError) as e:
```

Comment:
here I wonder if we should just catch Exception... there may be more and it is bound to change again in other polars releases

Reply:
Exception is a bit large ;). But maybe this is better.